examples/wifi-echo/server/esp32/main/wifi-echo.cpp

```diff
@@ -59,19 +59,17 @@ extern void startClient(void);
 
 #if CONFIG_DEVICE_TYPE_M5STACK
 
-#define BUTTON_1_GPIO_NUM GPIO_NUM_39               // Left button on M5Stack
-#define BUTTON_2_GPIO_NUM GPIO_NUM_38               // Middle button on M5Stack
-#define BUTTON_3_GPIO_NUM GPIO_NUM_37               // Right button on M5Stack
-#define STATUS_LED_GPIO_NUM GPIO_NUM_MAX            // No status LED on M5Stack
-#define LIGHT_CONTROLLER_OUTPUT_GPIO_NUM GPIO_NUM_2 // Use GPIO2 as the light controller output on M5Stack
+#define BUTTON_1_GPIO_NUM GPIO_NUM_39    // Left button on M5Stack
+#define BUTTON_2_GPIO_NUM GPIO_NUM_38    // Middle button on M5Stack
+#define BUTTON_3_GPIO_NUM GPIO_NUM_37    // Right button on M5Stack
+#define STATUS_LED_GPIO_NUM GPIO_NUM_MAX // No status LED on M5Stack
 
 #elif CONFIG_DEVICE_TYPE_ESP32_DEVKITC
 
-#define BUTTON_1_GPIO_NUM GPIO_NUM_34                // Button 1 on DevKitC
-#define BUTTON_2_GPIO_NUM GPIO_NUM_35                // Button 2 on DevKitC
-#define BUTTON_3_GPIO_NUM GPIO_NUM_0                 // Button 3 on DevKitC
-#define STATUS_LED_GPIO_NUM GPIO_NUM_2               // Use LED1 (blue LED) as status LED on DevKitC
-#define LIGHT_CONTROLLER_OUTPUT_GPIO_NUM GPIO_NUM_33 // Use GPIO33 as the light controller output on DevKitC
+#define BUTTON_1_GPIO_NUM GPIO_NUM_34  // Button 1 on DevKitC
+#define BUTTON_2_GPIO_NUM GPIO_NUM_35  // Button 2 on DevKitC
+#define BUTTON_3_GPIO_NUM GPIO_NUM_0   // Button 3 on DevKitC
```

Comment:
Another thing is, I see that the buttons are only used for M5Stack (under `CONFIG_HAVE_DISPLAY` option). If that's correct then we can remove the button macros for DevKitC and guard [this](https://github.com/project-chip/connectedhomeip/blob/master/examples/wifi-echo/server/esp32/main/wifi-echo.cpp#L108-L109) under `#if CONFIG_HAVE_DISPLAY`